airbyte-metrics/metrics-lib/src/main/java/io/airbyte/metrics/lib/MetricQueries.java

```diff
@@ -153,44 +153,67 @@ public static List<Pair<JobStatus, Double>> overallJobRuntimeForTerminalJobsInLa
    * are not matching with the number of expected runs according to the schedule settings. Refer to
```

Comment:
* A connection that is not running on schedule in last 24 hours is defined as a connection where the number of runs
   * are not matching with the number of expected runs according to the schedule settings. Refer to
   * playbook for detailed discussion.

Reply:
What does playbook refer to? Runbook?

Reply:
indeed. changed nae.